src/ray/protobuf/common.proto

```diff
@@ -630,19 +630,19 @@ message MetricPoint {
 // Type of a worker exit.
 enum WorkerExitType {
```

Comment:
maybe we can simplify it to:
SYSTEM_EXIT,
USER_EXIT,
SYSTEM_ERROR,
USER_ERROR

Reply:
EXIT == Intentional
ERROR == Unexpected

cc @jjyao when you read "Exit vs Error", did you feel it was clear?

Reply:
Actually, I did

INTENDED_SYSTEM_EXIT,
INTENDED_USER_EXIT,
SYSTEM_ERROR,
USER_ERROR

I think if I add INTENDED, it is clearer to understand the enum (since system exit usually includes unexpected exit iiuc)